numpy/core/arrayprint.py

```diff
@@ -708,9 +708,9 @@ def __call__(self, x):
             i = i + 'j'
         return r + i
 
+
 class DatetimeFormat(object):
-    def __init__(self, x, unit=None,
-                timezone=None, casting='same_kind'):
+    def __init__(self, x, unit=None, timezone='naive', casting='same_kind'):
```

Comment:
not sure what the standard for numpy is, but usually ``timezone=None`` is by definition naive.